CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.30.0
+
+**Release date:** 2023-02-16
+
+This prerelease comes with support for the new `ImagePolicy` v1beta2 API. In
+addition, the controller dependencies have been updated to their latest versions.
```

Comment:
Would maybe add a link here to the upgrade instructions around changing the image new, and a reminder about the deprecation of the flags.